### PR TITLE
load purge and refresh handlers by default

### LINKED ISSUE #183: Integrate Symfony HttpCache purge and refresh
Once https://github.com/FriendsOfSymfony/FOSHttpCache/pull/150 is merged, integrate the purge and refresh subscriber and document them.

### PR DESCRIPTION
**Pull request by dbu**
fix #183

### CODE REVIEW

**Comment by dbu** [comment]
oh wait, i will adjust the documentation as well.

**Comment by dbu** [comment]
hm, this makes me wonder: we do not have a proxy client for the symfony cache, and no configuration for it. for the user context, we did not need that, but for purge / refresh we need it. i guess we have to add that to the library and add some configuration here?

**Comment by ddeboer** [comment]
Yes, a separate SymfonyHttpCacheClient makes sense. See https://github.com/FriendsOfSymfony/FOSHttpCache/issues/192.

**Comment by dbu** [comment]
this needs https://github.com/FriendsOfSymfony/FOSHttpCache/issues/192 and then configuration for the client, and documentation updates.

**Comment by dbu** [comment]
hm, this reveals an issue with the supported versions of OptionsResolver. the foshttpcache is not properly testing its lowest dependencies.

**Comment by dbu** [comment]
ok so we wait for another version with https://github.com/FriendsOfSymfony/FOSHttpCache/pull/194 and https://github.com/FriendsOfSymfony/FOSHttpCache/pull/193

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "friendsofsymfony/http-cache": "~1.3,>=1.3.2",
+        "friendsofsymfony/http-cache": "~1.4@dev",
```
Comment (dbu) [inline composer.json]: needs to become ~1.4 when 1.4 is released. wait with merging until then.

File: Resources/config/nginx.xml [hunk 2]
```diff
@@ -14,6 +14,7 @@
             <argument>%fos_http_cache.proxy_client.nginx.servers%</argument>
             <argument>%fos_http_cache.proxy_client.nginx.base_url%</argument>
             <argument>%fos_http_cache.proxy_client.nginx.purge_location%</argument>
+            <argument type="service" id="fos_http_cache.proxy_client.nginx.guzzle_client" on-invalid="ignore"/>
```
Comment (dbu) [inline Resources/config/nginx.xml]: i think this is less surprising than what we had before with dynamically adding another argument in the DI extension class.

**Comment by dbu** [comment]
updated. this is ready for review and merge.

**Comment by ddeboer** [comment]
Released lib 1.4.0, so you can adjust dep.

**Comment by dbu** [comment]
and updated to use the stable release of foshttpcache. lets see what travis thinks.

**Pull request merged**